### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by joehan**
Switch Bug report template to use label 'Type: Bug' and adds 'Type: Feature Request' to feature request template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-name: ⚠️ Bug report
+name: "⚠️ Bug report"
```
Comment (bkendall) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: Why is it necessary to put this in quotes?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-name: 💡 Feature request
+name: "\U0001F4A1 Feature request"
```
Comment (bkendall) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: (I figure quotes are required here for the unicode character?)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (joehan) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Honestly, not sure it is necessary. I edited this via the Github UI and it automatically added those in. I can remove if you'd like, but I don't see any issue with including them.

**Review by bkendall — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bkendall) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: just my pet peeve. I don't understand why it defaults to adding quotes around strings that don't need to be quoted. w/e haha

**Pull request merged**